connect/api/src/main/java/org/apache/kafka/connect/data/ConnectSchema.java

```diff
@@ -230,7 +237,9 @@ public static void validateValue(Schema schema, Object value) {
             }
         }
         if (!foundMatch)
-            throw new DataException("Invalid Java object for schema type " + schema.type() + ": " + value.getClass());
+            throw new DataException("Invalid Java object for schema type " + schema.type()
```

Comment:
The error message is updated to say "required field". Do we actually know that this is a required field here?